### ISSUE TITLE
Loaded font for some languages keeps changing into the default font while typing

### ISSUE THREAD

**Issue by SweetChilli37** [photopea/photopea#2793]
Hi.

Mostly I type in English or Portuguese on a text layer but occasionally in Korean as well. This happened while I was tying in Korean with a loaded font specifically for Korean text but also available for English text. I can type first two or three words with the loaded font but then it suddenly changes into the default font (Noto Sans KR) for following words.

It never happens to English. By the way, it happens to Portuguese only when a font doesn't have the marks for the accents (ã á à ç for example). It's weird that it happens to Korean because it's free from the accentual marks and the font itself contains all formats for Korean language.

Thank you for you time!

**Comment by photopea** [comment]
Hi, this auto-switch should happen, when you insert characters not supported by the current font.

Are you sure, that it happens incorrectly, while your characters are supported by the font? Could you tell me, what characters and what font you used, so that I can reproduce it myself?

**Comment by SweetChilli37** [comment]
I'm sure it happened incorrectly. The font I downloaded is from a Korean site for free Korean font distribution so it supports both Korean and English language. I attached the font here but I'm afraid I don't really know every character that causes this issue. Aparrently, any sentence which starts with ㅂ, ㅊ, ㅁ, respectively corresponding to q, c, a of English character on keyboard, ends up escaping from the loaded font and going back to the default font. There can be more of them. By the way, there's no problem if I apply the font after the drag-select.

[BMEuljiro10yearslater.zip](https://github.com/photopea/photopea/files/5863142/BMEuljiro10yearslater.zip)

Thank you.
